package.json

```diff
@@ -12,24 +12,24 @@
     "babel-register": "^6.24.0",
     "coffee-script": "1.8.0",
     "dotenv": "^4.0.0",
+    "fs": "0.0.1-security",
```

Comment:
fs is built in library, no?

Reply:
I do know why, but when I run `hubot` locally, it asked me for `fs` library.

Reply:
I would try and debug this on your local machine. Don't think this is general bug and others need to install it as well

Reply:
@zelima COOOL! works fine, removed `fs`